src/main/java/com/yahoo/bullet/rest/service/StatusService.java

```diff
@@ -112,7 +110,7 @@ public void run() {
 
         String id = Utils.getNewQueryID();
         handlerService.addHandler(id, tickQueryHandler);
-        queryService.submit(id, TICK_QUERY);
+        queryService.submit(id, TICK_QUERY, null);
```

Comment:
Can we send the actual String? Probably be helpful to see ticks being logged in the backends

Reply:
The backend logs the actual query object. The string that gets sent just gets returned in the results metadata eventually.

Do you want to log both in the backend? (I want to at least see the query object logged)

Reply:
Oh does the backend log the actual query? I didn't notice. But since we have the RunningQuery and that has the BQL string, why don't we just log that and log the full query Object only in log level debug? I feel like it's a bit fat to log for every query, especially when replaying a million+ queries.